molgenis-file-ingester/src/main/java/org/molgenis/file/ingest/meta/FileIngestJobExecutionMetaData.java

```diff
@@ -22,7 +22,7 @@
 
 	public static final String URL = "url";
 	public static final String LOADER = "loader";
-	public static final List<String> LOADERS = Collections.singletonList("CSV");
+	public static final List<String> LOADERS = ImmutableList.of("CSV");
```

Comment:
Doesn't do anything, singletonList is also immutable!

Reply:
Sonar now recognizes that the List is immutable. I prefer this solution above marking the Sonar warning as false positive, because it might prevent issue detection is this code is changed.

Reply:
The issue was that the field should be protected

Reply:
The security issue is that the field should be protected, I think?
https://sonarcloud.io/project/issues?id=org.molgenis%3Amolgenis&issues=AVzJsld5sY0y2VuVC9S0&open=AVzJsld5sY0y2VuVC9S0

Reply:
Yes, because Sonar thinks it is a mutable field: see the '...' in Sonar for details.

Reply:
Ah, I see!

Reply:
For clarity we could store Immutable* values in a Immutable* instead of *, but lets make that a different task.